.github/workflows/ci.yml

```diff
@@ -6,17 +6,26 @@ jobs:
 # whenever you do end up solving this issue so that I know how many years have passed.
 # Until that time, please manually run the test suite before pushing. Thanks. 
 #   - ValentijnvdBeek
```

Comment:
The comment here is obsolete so it would be nice if you where to delete it.

Reply:
@ValentijnvdBeek I've just removed it.